graphql/types/Item.graphql

```diff
@@ -45,6 +45,7 @@ type Item {
   uniqueId: String!
   unitMultiplier: Float
   priceTags: [PriceTag!]!
+  isGift: Boolean
```

Comment:
the `isGift` property is required, so we can type it as non-nullable

```suggestion
  isGift: Boolean!
```

Reply:
Done!